src/collectd.conf.pod

```diff
@@ -3318,6 +3318,18 @@ at most B<16384> to prevent typos and dumb mistakes.
 
 =back
 
+=head2 Plugin C<epics>
+
+=over 4
+
+=item B<Variable> I<PVName>
+
+=item B<Type> I<Type>
+
+=item B<Label> I<PVName>
```

Comment:
I'm not sure I'd be able to use the plugin based on this. Maybe copy the example from `collectd.conf.in` here as a "synopsis" and describe what effect that configuration would have? E.g. what would happen with the "label"?